### ISSUE TITLE
Skype doesn't open link

### ISSUE THREAD

**Issue by nicolangr** [saenzramiro/rambox#433]
When click on link in chat the browser dosn't open it.

## 

> Rambox 0.4.4
> Electron 1.4.1
> linux x64 3.19.0-32-generic

**Comment by saenzramiro** [comment]
Did you follow the steps in the Wiki?

https://github.com/saenzramiro/rambox/wiki/Skype#links

**Comment by Hasimir0** [comment]
Yes I did, but to no avail.
More or less one month ago, I just needed to left click a link to have it load in the main browser.
Since then, all I can do is right click the link, select the "copy as text" option and manually glue the web address into a new browser tab.

Turning ON the Web Link Preview translates some (not all) links into a preview box, that I can right click to access the "Open" option.
But it still not the same as just left clicking the link (or its preview) and have it pop up open in a new browser tab.

Help? :)
